src/main/webapp/util/script.js

```diff
@@ -14,9 +14,20 @@ export function getBuildDirPath() {
     return '';
 }
 
-export function loadGlobalConfig() {
-    // Get the configuraiton json file in sync mode
+function loadJSONFile() {
     return axios.get(`${getBuildDirPath()}/globalConfig.json`).then((res) => {
         return typeof res.data === 'object' ? res.data : JSON.parse(res.data);
     });
 }
+
+function loadYAMLFile() {
+    const yaml = require('js-yaml');
+    return axios.get(`${getBuildDirPath()}/globalConfig.yaml`).then((res) => {
+        return typeof res.data === 'object' ? res.data : yaml.load(res.data);
+    });
+}
+
```

Comment:
Is it possible to add a unit test for this behaviour? It seems to be quite isolated from the source code, so should not very complicated to do.

Reply:
The front-end code does not have a unit test. I intend to include some test cases in the backend code.